sdk/examples/client/01_list_addresses.rs

```diff
@@ -1,9 +1,14 @@
 // Copyright 2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-//! TODO: Example description
+//! This example shows how to list the generated addresses of an account.
```

Comment:
It also generates them in the first place, descriptions sounds like it's copied from the wallet example